scripts/commands/checkMetadata.ts

```diff
@@ -10,14 +10,34 @@
 // copyright notice, and modified files need to carry a notice indicating
 // that they have been altered from the originals.
 
-import grayMatter from "gray-matter";
 import fs from "fs/promises";
+
+import yargs from "yargs/yargs";
+import { hideBin } from "yargs/helpers";
+import grayMatter from "gray-matter";
 import { globby } from "globby";
 
-const IGNORED_FILES = new Set([
-  "docs/api/qiskit/transpiler_builtin_plugins.md",
-  "docs/api/qiskit/dev/transpiler_builtin_plugins.md",
-]);
+interface Arguments {
+  [x: string]: unknown;
+  apis: boolean;
```

Comment:
Nit: Add the translations argument
```suggestion
  apis: boolean;
  translations: boolean;
```

Reply:
Thanks!